api/src/test/scala/dcos/metronome/api/WithApplicationComponents.scala

```diff
@@ -121,6 +122,8 @@ class MockApiComponents(context: Context) extends BuiltInComponentsFromContext(c
 
   lazy val assets: Assets = wire[Assets]
 
+  lazy val queueService: QueueService = QueueServiceFixture.simpleQueueService()
```

Comment:
what is the motivation behind this change? I see it is never used ... ? I don't see much value in such fixture, maybe we can drop it ... ?

Reply:
it was focused on me for the `wire[ApiModule]` below.  Doesn't compile without it :)